StRoot/StMuDSTMaker/COMMON/StMuFcsHit.h

```diff
@@ -28,6 +28,11 @@ class StMuFcsHit : public TObject {
              unsigned short ns, unsigned short ehp, unsigned short dep, unsigned short ch, 
              float e);
     ~StMuFcsHit();
+
+    virtual void Clear (Option_t * opt=""){
+        TObject::Clear(opt);
```

Comment:
TObject::Clear() seems to be empty... At least 
https://root.cern.ch/doc/master/classTObject.html#a0ccdc7dd6ee02ee79c46bfab0e1be291
shows only links to reimplementations. I don't know what this line do.

Reply:
good point, it is not needed. I will remove